test/skipped/advi_demo.jl

```diff
@@ -0,0 +1,129 @@
+using Turing
```

Comment:
I don't think it necessary to have the demo as a test case. Maybe replace this with unit-tests.

Reply:
This was suggested by @xukai92, so I guess he'll have to chime in here. I was under the understanding that this was supposed to be "non-unit tests" and only used as a small script to demo the algo, hence why they're kept in the `skipped` folder.

I do have a simple unit-test in `tests/inference/vi.jl` btw (which should be separated into a `variational` one when if we agree on this submodule-structure).

Reply:
I see. I didn't see it was skipped.